monitoring/configuration/sa-rbac-proxy

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kubestellar-controller-manager
```

Comment:
Mismatch between this name and the file name.

Reply:
Removed this file - as it's not needed.